public/scripts/openai.js

```diff
@@ -160,12 +155,6 @@ function setOpenAIMessages(chat) {
             content = `${chat[j].name}: ${content}`;
         }
 
-        // system messages produce no content
-        if (chat[j]['is_system']) {
-            role = 'system';
-            content = '';
-        }
-
```

Comment:
Dead code even before this PR, because `Generate()` was making sure that system messages never reach this function.